apps/frontend/src/pages/Profile.tsx

```diff
@@ -0,0 +1,264 @@
+import {
+  Avatar,
+  Box,
+  Button,
+  TextField,
+  Typography,
+  useMediaQuery,
+  useTheme,
+} from "@mui/material";
+import AccountSidebar from "../components/AccountSidebar";
+import { useStyles } from "../styles/pageStyles";
+import { useEffect, useState } from "react";
+import _axios_instance from "../_axios_instance";
+import { useParams } from "react-router-dom";
+
+interface ImageURLObject {
+  url: string;
+}
+
+const Profile = () => {
+  const classes = useStyles();
+  const theme = useTheme();
+  const isDesktop = useMediaQuery(theme.breakpoints.up("sm"));
+  const isScreenSmall = useMediaQuery(theme.breakpoints.down("lg"));
```

Comment:
I don't like these names, desktop would normally be considered to be lg, and a variable named isScreenSmall should be checking the small (sm) breakpoint, not lg